src/Vezeeta.Domain/Entities/DiscountCode.cs

```diff
@@ -0,0 +1,27 @@
+﻿
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Vezeeta.Domain.Enums;
+
+namespace Vezeeta.Domain.Entities
+{
+    public class DiscountCode
```

Comment:
For all entities, you should be inheriting from `BaseEntity<T>` where T is the datatype of the ID for your entity.